lib/response.js

```diff
@@ -32,7 +32,7 @@ var generate = function(req, res, next) {
 
             } else {
 
-                var size = typeof service.size === 'function' ? service.size.apply(service, [params, query, body, cookies]) : service.size;
+                var size = _.isFunction(service.size) ? service.size.apply(service, [params, query, body, cookies]) : service.size;
```

Comment:
this change is not related to the actual feature PR, but it was just to stick to the lodash test pattern.